.travis.yml

```diff
@@ -1,23 +1,20 @@
-dist: trusty
-sudo: required
+dist: bionic
 language: go
 go:
-    - 1.10.x
-    - 1.11.x
+    - 1.13.x
 
 install:
-    - go get -d
-    - go get -u github.com/vbatts/git-validation
-    - go get -u github.com/kunalkushwaha/ltag
+    - go mod vendor
+    - pushd ..; go get -u github.com/vbatts/git-validation; popd
+    - pushd ..; go get -u github.com/kunalkushwaha/ltag; popd
 
 before_script:
     - pushd ..; git clone https://github.com/containerd/project; popd
 
 script:
     - DCO_VERBOSITY=-q ../project/script/validate/dco
     - ../project/script/validate/fileheader ../project/
-    - go test -race -covermode=atomic -c
-    - sudo ./aufs.test -test.root -test.v -test.coverprofile=coverage.txt
+    - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
```

Comment:
this part broke the CI, @chenrui333 could you revert this part?

```
=== RUN   TestAufs
--- SKIP: TestAufs (0.00s)
    helpers_unix.go:41: skipping test that requires root
```

Reply:
Got it, I will fix it now!

Reply:
PR, https://github.com/containerd/aufs/pull/19